### ISSUE TITLE
COMPRESSION_OPTIONS for zstd can't be customized

### ISSUE THREAD

**Issue by Ropid** [archlinux/mkinitcpio#40]
In the `mkinitcpio` script, there's this line here that sets default values for `$COMPRESSION_OPTIONS` for zstd:

https://github.com/archlinux/mkinitcpio/blob/6e4603638b459468d235437c6a47584d609a86a2/mkinitcpio#L215

This makes it so you can't customize the compression level in `/etc/mkinitcpio.conf`. If you for example try to do this here in the config file:

`COMPRESSION_OPTIONS=( -10 -T0 )`

Then later mkinitcpio will run the zstd process with this command line:

`zstd -10 -T0 -19 -T0`

The default `-19` will then override the `-10` in the config.

Thinking about what to do, instead of using `+=` to add to the back of the variable, you could add the defaults in front of the user's settings like this:

```
COMPRESSION_OPTIONS=( '-19' '-T0' "${COMPRESSION_OPTIONS[@]}" )
```

An `/etc/mkinitcpio.conf` value could then override the default `-19`.

**Comment by Maryse47** [comment]
Duplicate of https://github.com/archlinux/mkinitcpio/pull/39

**Comment by pfactum** [comment]
https://bugs.archlinux.org/task/68470

**Comment by grazzolini** [comment]
Merged #39
